test/acceptance/cli-test.js

```diff
@@ -704,6 +737,29 @@ Options:
     });
   }
 
+  function setProjectConfigForErrorsWithIgnoredDir() {
```

Comment:
Let’s just move this inline into the specific test that uses it (I generally prefer to keep the input and the expectations together such that reviewing the test is easier without jumping around too much), especially since it isn’t used a bunch of times like some of the others.

Reply:
Sure thing, unrolled the setup